src/Dialogs/VolumePropertiesWindow.vala

```diff
@@ -97,7 +97,7 @@ public class VolumePropertiesWindow : AbstractPropertiesDialog {
         }
 
         create_storage_bar (info, 3);
-        update_storage_block_size (used_space, Files.StorageBar.ItemDescription.FILES);
+        storage_levelbar.value = used_space;
```

Comment:
You should not update the storage bar total size here now - it results with the storage bar only showing the selection size, overriding the desired filesystem used + reserved.